nipype/interfaces/mrtrix3/utils.py

```diff
@@ -463,7 +479,8 @@ class ComputeTDIInputSpec(CommandLineInputSpec):
         "(these lengths are then taken into account during TWI calculation)",
     )
     ends_only = traits.Bool(
-        argstr="-ends_only", desc="only map the streamline" " endpoints to the image"
+        argstr="-ends_only",
+        desc="only map the streamline" " endpoints to the image",
```

Comment:
While we're modifying this line...

```suggestion
        desc="only map the streamline endpoints to the image",
```